inst/include/rcpp_sink.h

```diff
@@ -29,7 +29,12 @@ class r_sink : public base_sink<Mutex> {
         // sending it to its final destination:
         spdlog::memory_buf_t formatted;
         spdlog::sinks::base_sink<Mutex>::formatter_->format(msg, formatted);
+
+        #ifdef SPDLOG_USE_STD_FORMAT
+        Rcpp::Rcout << formatted;
+        #else
         Rcpp::Rcout << fmt::to_string(formatted);
+        #endif
```

Comment:
This needs a line or two of comment before line 34 to refer to the proposed change in formatter.cpp which ensures (as a side-effect here) that what is passed to Rcpp::Rcout is now a string.  I am also scratching my head over whether is would be clearer to access that change via wrapper here, just like `fmt::to_string()` does.